docs/user/implement_env.md

```diff
@@ -0,0 +1,308 @@
+# Adding a New Environment
+
+Garage uses the [`Environment`](https://github.com/rlworkgroup/garage/blob/master/src/garage/_environment.py)
+API for all RL environments.
+
+The public API methods of :`Environment` are:
+
+* `reset()`: Resets the environment. Return the first observation, and the episode-level information.
```

Comment:
Please use tables for these sections as you did in the docstring. You can refer directly to the class reference using e.g.```:meth:`~Environment.reset` ``` which should render as "reset" with a link to the API docs.

Reply:
:meth:`~Environment.reset` doesn't seem to work, even in code strings. Eg :method:`~LocalRunner.step_epochs()` in the `train` docstring doesn't link to the API reference in the latest read the doc [page](https://garage.readthedocs.io/en/latest/_autoapi/garage/torch/algos/index.html#id1).

And sphix doesn't come with support of markdown table (https://github.com/readthedocs/readthedocs.org/issues/4466). Though there seem to be some workarounds but the setup could be a pain. We can either:
1. stick to bullet points 2. Try set up markdown table support (https://pypi.org/project/sphinx-markdown-tables/).

What do you think?

Reply:
``:meth: `~LocalRunner.step_epochs()` `` works for me in `train()`

Reply:
I see. Yea this seems to only work in docstrings but not in rst/md files.